server/container.go

```diff
@@ -597,5 +597,27 @@ func (s *Server) PortForward(ctx context.Context, req *pb.PortForwardRequest) (*
 
 // Status returns the status of the runtime
 func (s *Server) Status(ctx context.Context, req *pb.StatusRequest) (*pb.StatusResponse, error) {
-	return nil, nil
+
+	// Deal with Runtime conditions
+	runtimeReadyConditionString := "RuntimeReady"
+	runtimeReady, err := s.runtime.RuntimeReady()
+	if err != nil {
+		return nil, err
+	}
+	networkReadyConditionString := "NetworkReady"
```

Comment:
It's better to just use `pb.RuntimeReady` and `pb.NetworkReady` instead of defining new strings here.

They are already vendored at [here](https://github.com/kubernetes-incubator/cri-o/blob/master/vendor/src/k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime/constants.go).